src/tools/tests/schema2graph-test.ts

```diff
@@ -34,14 +34,32 @@ function convert(graph: SchemaGraph) {
   return {nodes, aliases};
 }
 
+function dummyNameGenerator(node: SchemaNode, i: number) {
+  if (i === -1) {
+    return `${node.particleName}_${node.connections[0]}`;
+  }
+  return `${node.particleName}Internal${i}`;
+}
+
+function dummyAliasGenerator(node: SchemaNode): string[] {
+  const arr: string[] = [];
+  if (node.connections.length === 1) {
+    return arr;
+  }
+  for (const connection of node.connections) {
+    arr.push(`${node.particleName}_${connection}`);
+  }
+  return arr;
+}
```

Comment:
These don't really look like dummy implementations to me... Can you reuse the real implementations? Or otherwise, it's probably better to make them really dumb, e.g. `return "name${i}";`